packages/postcss/package.json

```diff
@@ -19,12 +19,15 @@
   },
   "dependencies": {
     "css-loader": "^3.2.0",
+    "core-js": "^3.6.2",
```

Comment:
In https://github.com/xing/hops/blob/master/packages/jest-preset/package.json we have `core-js` as `^3.2.1`, should we align these versions so that they're deduplicated in the yarn.lock?

Reply:
Go for it 👍